action.yml

```diff
@@ -0,0 +1,76 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
+  network:
+    description: "Network to run"
+    default: "local"
+  enable_logs:
+    description: "Boolean flag enabling the logs"
+    default: "true"
```

Comment:
Could also include the `enable` param, which is a comma-separated-list of services that run.